### PR TITLE
Add scripts for finding Landsat scenes for the discovery on Lakes

### PR DESCRIPTION
**Pull request by j08lue**
To fulfill https://github.com/NASA-IMPACT/veda-data-pipelines/issues/356, we need first to identify suitable Landsat scenes that cover the individual lakes.

The attached scripts do that and generate STAC collection JSON and a list of scenes to ingest.

All hacked together pretty quickly, but it worksTM.

Subsequently, we should add any scripts we use to generate the STAC items (copying them from Landsat PDS STAC, I guess).

### CODE REVIEW

**Comment by j08lue** [comment]
I see we are now getting code in for creating complete STAC metadata - thanks, @ividito.

We need to make a couple of changes before the data looks like the collections we already have - for example https://staging-stac.delta-backend.com/collections/landsat-c2l2-sr-antarctic-glaciers-thwaites/items/LC08_L2SR_005113_20221220_20221227_02_T2_SR.

## Required changes

1. Load STAC item metadata from Landsat PDS (AWS) instead of Planetary Computer, so they contain the Landsat PDS bucket URLs
2. Move the S3 URLs out of the `alternate_assets` STAC extension and into the item asset href
3. Clean up: Remove all other bands than RGB and remove mentioning of the `alternate_assets` extension

Would you be up for making these changes, @ividito?

**Comment by ividito** [comment]
I was getting request rate blockers from `landsatlook`, so I took a bit of a different approach to get the right asset links added. The new collections should be updated on staging now.

File: transformation-scripts/landsat-lakes-discovery/aral-sea.ipynb [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c55df1ce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pypgstac load items vanern-items.json --dsn postgresql://$DB_USERNAME:$DB_PASSWORD@$DB_HOSTNAME:5432/postgis --method upsert"
```
Comment (j08lue) [inline transformation-scripts/landsat-lakes-discovery/aral-sea.ipynb:383]: Copy-paste glitch: This should be `aral-sea-collection.json`.

**Comment by j08lue** [comment]
I think we can just merge this now, @ividito. Thanks for the good work!

**Review by jsignell — APPROVED** [review]
This generally looks fine to me. I think if possible it would be better to just have a pre-defined query and access the items from the canonical storage location, but I guess that is blocked by https://github.com/NASA-IMPACT/veda-architecture/issues/294

**Comment by j08lue** [comment]
> I think if possible it would be better to just have a pre-defined query and access the items from the canonical storage location, but I guess that is blocked

That is not blocked and would definitely have been a better pattern. I'll keep it as-is now, but we should strive to make end-to-end notebooks in the future.

**Pull request merged**